managed-databases/postgresql-and-mysql/api-cli/verify-ca-postgresql.mdx

```diff
@@ -0,0 +1,256 @@
+
+In this page we show you two types of protection modes you can use to verify if the servers connecting to your client are properly encrypted.
+
+We also show you how to run the verification using different languages, such as `psql`, `python`, `go`, and `js`.
+
+## Using verify-ca
+
+Will verify that a server is trustworthy by checking the certificate chain up to the root certificate stored on the client.
+
+### With psql
+
+```sql
+PGSSLROOTCERT=<SSL_CERTIFICATE> psql "host=<IP> port=<PORT> user=<USERNAME> sslmode=verify-ca dbname=<DB_NAME>"
+```
+
+### With Python
+
+```python
+import psycopg2
+
+connectionInfos = {
+    'host': <IP or HOSTNAME>,
+    'port': <PORT>,
+    'user': <USERNAME>,
+    'password': <PASSWORD>,
+    'sslmode': 'verify-ca',
+    'sslrootcert': <PATH_TO_SSL_CERTIFICATE>,
+    'dbname': <DB_NAME>
+}
+
+try:
+    connection = psycopg2.connect(**connectionInfos)
+    print("connected successfully")
+except psycopg2.Error as e:
+    print(e)
+```
+
+### With Go
+
+```go
+import (
+  "database/sql"
+  "fmt"
+
+  _ "github.com/lib/pq"
+)
+
+const (
+  host          = <IP or HOSTNAME>
+  port          = <PORT>
+  user          = <USERNAME>
+  password      = <PASSWORD>
+  dbname        = <DB_NAME>
+  sslrootcert   = <PATH_TO_SSL_CERTIFICATE>
+  sslmode       = 'verify-ca'
+)
+
+func main() {
+  psqlInfo := fmt.Sprintf("host=%s port=%d user=%s "+
+    "password=%s dbname=%s sslmode=%s sslrootcert=%s" ,
+    host, port, user, password, dbname, sslmode, sslrootcert)
+  db, err := sql.Open("postgres", psqlInfo)
+  if err != nil {
+    panic(err)
+  }
+  defer db.Close()
+
+  err = db.Ping()
+  if err != nil {
+    panic(err)
+  }
+
+  fmt.Println("Successfully connected!")
+}
+```
+
+### With JS
+
+```js
+import fs from 'fs'
+
+const config = {
+    user: <USERNAME>,
+    password: <PASSWORD>,
+    database: <DB_NAME>,
+    port: <PORT>,
+    host: <IP or HOSTNAME>,
+    ssl: {
+        rejectUnauthorized: false,
+        cert: fs.readFileSync(<PATH_TO_SSL_CERTIFICATE>).toString(),
+        mode: 'verify-ca'
+    },
+}
+import pkg from 'pg';
+const { Client, Pool } = pkg;
+
+const client = new Client(config)
+client.connect(err => {
+    if (err) {
+        console.error('error connecting', err.stack)
+    } else {
+        console.log('connected')
+        client.end()
+    }
+})
+
+const pool = new Pool(config)
+pool
+    .connect()
+    .then(client => {
+        console.log('connected')
+        client.release()
+    })
+    .catch(err => console.error('error connecting', err.stack))
+    .then(() => pool.end())
+```
+
+
+## Using verify-full
+
+Will verify that a server is trustworthy by checking the certificate chain up to the root certificate stored on the client and if the hostname specified in the server certificate matches the hostname of the server.
```

Comment:
same here + break the sentence into two 

```suggestion
This option will verify that a server is trustworthy by checking the certificate chain up to the root certificate stored on the client. It will also verify whether the hostname specified in the server certificate matches the hostname of the server.
```